backend-worker/src/main/kotlin/org/rucca/cheese/auth/AuthorizationService.kt

```
@@ -0,0 +1,171 @@
+        if (token.indexOf("Bearer ") == 0) tokenWithoutBearer = token.substring(7)
+        else if (token.indexOf("bearer ") == 0) tokenWithoutBearer = token.substring(7)
+
+        val payload: TokenPayload
+        try {
+            payload =
+                objectMapper.readValue(
+                    verifier.verify(tokenWithoutBearer).getClaim("payload")?.toString()
+                        ?: throw InvalidTokenError(),
+                    TokenPayload::class.java,
+                )
+        } catch (e: TokenExpiredException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)